lib/io/test/io.test.js

```diff
@@ -4,9 +4,20 @@ import {
   loadNotes,
   deleteNote,
   savePreferences,
-  loadPreferences
+  loadPreferences,
+  updatePreferences
 } from "../index.js";
 import { noteExistsIn, preferences, newNote } from "./util.js";
+import path from "path";
+import tmp from "tmp";
+tmp.setGracefulCleanup();
```

Comment:
Why do we need a tmp directory? The tests save stuff in the .aurora/test directory already.